src/nft/components/collection/ActivityCells.tsx

```diff
@@ -57,6 +58,15 @@ const AddressLink = styled(ExternalLink)`
   }
 `
 
+const isPurchasableOrder = (orderStatus?: OrderStatus, marketplace?: string): boolean => {
+  if (!marketplace || !orderStatus) return false
+  const purchasableMarkets = Object.keys(NftMarketplace).map((market) => market.toLowerCase())
+
+  const validOrder = orderStatus === OrderStatus.Valid
+  const purchasableMarket = purchasableMarkets.includes(marketplace.toLowerCase())
```

Comment:
nit: naming it `purchasableMarket` when `purchasableMarkets` is an array of strings implies this would be a string too, but i think it's a boolean